src/Extensions/Hosting.Services.Web/Middlewares/ResponseHeadersMiddleware.cs

```diff
@@ -12,7 +12,10 @@ namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
 	/// </summary>
 	public class ResponseHeadersMiddleware
```

Comment:
Same question, since this is public and can be consumed by projects which don't use the new flag, why do the methods not have null guards?

Reply:
Solved here https://github.com/microsoft/Omex/pull/181#discussion_r400235016